src/pages/Checkout/CheckoutPage.vue

```diff
@@ -645,19 +645,12 @@ export default {
 				query: shopBasketUpdate,
 				fetchPolicy: 'network-only',
 			}).then(({ data }) => {
-				// when updating basket state, check for free credits and redirect if present
+				// when updating basket state, check for free credits and remove guest checkout option
 				const hasFreeCredits = _get(data, 'shop.basket.hasFreeCredits');
-				if (hasFreeCredits) {
-					if (refreshEvent === 'kiva-card-applied') {
-						this.$kvTrackEvent('basket', 'free credits applied', 'exit to legacy');
-						this.disableGuestCheckout();
-					}
-					this.redirectLightboxVisible = true;
```

Comment:
Can `redirectLightboxVisible` and the associated lightbox be removed? I'm not seeing that flag being set to `true` anymore in this file.

Reply:
Yea good call. Was also just looking at that.

There is still one redirect case but we're handling it directly in the apollo prefetch